build.gradle

```diff
@@ -140,15 +140,19 @@ repositories {
     }
 
     maven {
-        url = uri('https://jfrog.sandbox.fynarfin.io/artifactory/fyn-libs-snapshot')
+        url = uri('https://fynarfinhub.jfrog.io/artifactory/ph-ee-repo-local')
+        credentials {
+            username = "maven-repo-user"
+            password = "123QWEasd"
+        }
     }
 }
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter'
     testImplementation "com.github.tomakehurst:wiremock-jre8:2.35.0"
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
-    implementation 'org.mifos:ph-ee-connector-common:1.8.1-SNAPSHOT'
+    implementation 'org.mifos:ph-ee-connector-common:0.0.0'
```

Comment:
Also, update this to the released version after the jfrog issue is fixed.

Reply:
okay